components/layout/display_list/mod.rs

```diff
@@ -1662,13 +1676,23 @@ fn glyphs(
                     index: glyph.id(),
                     point,
                 };
-                glyphs.push(glyph);
+
+                // First glyph must never be ellided. otherwise, check if it's time to crop.
```

Comment:
It's spelled `elided`, not `ellided`. Needs to be fixed here, in `can_be_ellided`, and so on.